gammapy/data/pointing.py

```diff
@@ -25,6 +27,22 @@
 __all__ = ["FixedPointingInfo", "PointingInfo", "PointingMode"]
 
 
+def _check_coord(coord_or_frame, expected_frame, name):
+    is_coord = isinstance(coord_or_frame, SkyCoord)
+    is_frame = isinstance(coord_or_frame, BaseCoordinateFrame)
+
+    if not (is_frame or is_coord):
+        raise TypeError(f"{name} must be a {SkyCoord} or {expected_frame} instance")
```

Comment:
{SkyCoord} -> `SkyCoord` ?

Reply:
Using `{SkyCoord}` will give the full import path:

```
In [3]: f"{SkyCoord}"
Out[3]: "<class 'astropy.coordinates.sky_coordinate.SkyCoord'>"
```

Reply:
Which might be a bit too verbose... will change